### ISSUE TITLE
Idea: How about a Manual Animated Portrait rather than Automated.

### ISSUE THREAD

**Issue by cyliamargatroid** [Windblade-GR01/Ikemen-GO#77]
Like MUGEN Hook, but instead of that, it uses DEF Files
```
[anim-port]
; Idle, P2 Idle, Select, P2 Select
anim = 0 0 180 180; Reads Animations
```
and in Screenpack, its almost setting up Portraits, but uses portrait.anim

**Comment by Windblade-GR01** [comment]
That's the plan.
It's very similar to what I had planned.
```
portraitAnim = 0, 15, 50, 975
```

**Comment by cyliamargatroid** [comment]
Pretty much like Almost MugenHook.

**Comment by Orochikyocr** [comment]
This will be really useful, using character anim in system.def for portraits in both select and versus, also for small select screen portraits.

**Comment by K4thos** [comment]
this is already implemented since 0.96
